paddle/fluid/operators/elementwise/mkldnn/elementwise_add_mkldnn_op.cc

```diff
@@ -51,12 +51,14 @@ class EltwiseAddMKLDNNKernel : public framework::OpKernel<T> {
     auto y_dims_untrimed = y->dims();
     auto z_dims = z->dims();
 
+    mkldnn::stream astream(mkldnn_engine);
+
     // Execute default elementwise_add operator when
     // broadcast operations need to performed.
     if (x_dims != y_dims_untrimed) {
       Tensor _x;
       MKLDNNMemoryFormat format;
-      std::vector<int> src_x_tz = framework::vectorize<int>(x_dims);
+      auto src_x_tz = framework::vectorize(x_dims);
```

Comment:
Why vectorize<int> -> vectorize ?

Reply:
It's because vectorize<int>() returns std::vector<int> which is needed in MKL-DNN 0.2 kernels but vectorize() returns std::vector<int64_t> needed by MKL-DNN 1.0 kernels.